packages/browser/src/plugins/ajs-destination/index.ts

```diff
@@ -267,6 +268,7 @@ export class LegacyDestination implements DestinationPlugin {
       ctx.stats.increment('analytics_js.integration.invoke.error', 1, [
         `method:${eventType}`,
         `integration_name:${this.name}`,
+        `type:classic-destination`,
```

Comment:
We should just add "recordMetric" to the stats class or have it as a util that we can use here?

Reply:
I thought of that too, but I'm not 100% sure of where to put it. One side of me saw Stats class as agnostic of its usage.

For now - we can leave it as-is, since we need to merge this soon.

Reply:
just put it in a utility function?